server/app/uk/gov/ons/addressIndex/server/modules/AddressIndexRepository.scala

```diff
@@ -126,7 +124,7 @@ class AddressIndexRepository @Inject()(
             matchQuery(
               field = "lpi.paoStartNumber",
               value = token
-            )),
+            ).boost(queryParams.buildingNumberLpiBoost)),
           tokensMap.get(Tokens.locality).map(token =>
```

Comment:
I don't think we need a configurable boost on Locality at this point. We need to modify the query anyway after we have re-indexed.